requirements.txt

```diff
@@ -3,7 +3,7 @@ PySerial>=2.7
 PrettyTable>=0.7.2
 Jinja2>=2.7.3
 IntelHex>=1.3
-project-generator>=0.9.3,<0.10.0
+project-generator>=0.9.6,<0.10.0
```

Comment:
I would propose the latest v0.9.7 as 0.9.6 contained a bug with the iar template.